low-level/files/opt/seagate/sspl/bin/conf_sensors_enable

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -e -u -o pipefail
+
+to_lower_case() {
+    tr '[:upper:]' '[:lower:]'
+}
+
+declare -a sed_expr=(-e '')
+storage_type="$(provisioner get_setup_info --output json | jq -r .ret.storage_type | to_lower_case)"
+server_type="$(provisioner get_setup_info --output json | jq -r .ret.server_type | to_lower_case)"
+
+declare -A should_monitor=(
+    [REALSTORSENSORS]="true"
+    [NODEHWSENSOR]="true"
+    [SYSTEMDWATCHDOG]="true" # This is the 'server disks' sensor
+    [RAIDSENSOR]="true"
+    [SASPORTSENSOR]="true"
+    [MEMFAULTSENSOR]="true"
+    [CPUFAULTSENSOR]="true"
+
+)
+
+case "${storage_type}" in
+    virtual) 
+        should_monitor[REALSTORSENSORS]="false"
```

Comment:
should_monitor[] -> monitor[] can we keep, more simple

Reply:
Done.